test/api.js

```diff
@@ -19,6 +19,21 @@ describe('client-side api', function () {
 				.call(done);
 		});
 
+		it('should ensure that duplicate bundleName-messageName pairs are ignored', done => {
+			e.apis.extension.listenFor('testMessageName', 'testBundleName', done);
+			e.apis.extension.listenFor('testMessageName', 'testBundleName', done);
+			assert.equal(1, e.apis.extension._messageHandlers.length);
+			done();
+		});
+
+		it('should produce an error if a callback isn\'t given', () => {
+			try {
+				e.apis.extension.listenFor('testMessageName', 'test');
+			} catch (err) {
+				assert.equal(err.message, 'argument "handler" must be a function, but you provided a(n) string');
+			}
+		});
```

Comment:
Can you change this to use `assert.throws` instead? I find that to be a bit clearer. [See here](https://github.com/nodecg/nodecg/blob/master/test/schemas.js#L424-L432) for an example.